apps/sms/index.html

```diff
@@ -127,6 +128,7 @@
   </head>
 
   <body role="application" class="skin-comms theme-communications">
+    <iframe src="/views/shared/shim_host.html" class="shim-host" hidden></iframe>
```

Comment:
Set the iframe hidden seems a little better based on profiling result: https://docs.google.com/a/mozilla.com/spreadsheets/d/1u8u7OswOxHD8t4WgjaOrfzdMYx6bkOz3CahxHUYX7Gw/edit?usp=sharing